packages/kosu-contract-helpers/src/index.ts

```diff
@@ -0,0 +1,14 @@
+export { Kosu } from "./Kosu";
+
+export { EventEmitter } from "./EventEmitter";
+export { KosuToken } from "./KosuToken";
+export { OrderGateway } from "./OrderGateway";
+export { OrderHelper } from "./OrderHelper";
+export { PosterRegistry } from "./PosterRegistry";
+export { Signature } from "./Signature";
+export { Treasury } from "./Treasury";
+export { NULL_ADDRESS, toBytes32 } from "./utils";
+export { ValidatorRegistry } from "./ValidatorRegistry";
+export { Voting } from "./Voting";
+export { OrderSerializer } from "./OrderSerializer";
+export { NodeClient } from "./NodeClient";
```

Comment:
I think this should be moved to its own package... This is something I can do this week.

Reply:
I added a WIP tag to be more clear.  The only package that resembles its final state is kosu.js, though it will be undergoing changes as more components are separated into individual packages.